.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,123 @@
+name: 🐛 Bug Report
+description: Report a bug that affects a Material Web component
+
+body:
+  - type: dropdown
+    id: affected
+    validations:
+      required: true
+    attributes:
+      label: What is affected?
+      multiple: true
+      description: Note you can select more than one.
+      options:
```

Comment:
these options seem fine to me rn, though it might be annoying to maintain them

Reply:
I can remove this question since it is a bit long, or change it to more general themes like:

 - component
 - theming
 - accessibility
 - tooling
 - other

Reply:
I like the more general options!

Reply:
Done